raiden/messages.py

```diff
@@ -610,23 +647,24 @@ def pack(self, packed) -> None:
 
 @dataclass(repr=False, eq=False)
 class LockedTransfer(LockedTransferBase):
-    """
-    A LockedTransfer has a `target` address to which a chain of transfers shall
-    be established. Here the `secrethash` is mandatory.
-
-    `fee` is the remaining fee a recipient shall use to complete the mediated transfer.
-    The recipient can deduct his own fee from the amount and lower `fee` to the remaining fee.
-    Just as the recipient can fail to forward at all, or the assumed amount,
-    it can deduct a too high fee, but this would render completion of the transfer unlikely.
-
-    The initiator of a mediated transfer will calculate fees based on the likely fees along the
-    path. Note, it can not determine the path, as it does not know which nodes are available.
-
-    Initial `amount` should be expected received amount + fees.
-
-    Fees are always payable by the initiator.
-
-    `initiator` is the party that knows the secret to the `secrethash`
+    """ Message to used to reserve tokens for a new mediated transfer.
+
+    For this message to be valid, the sender must:
+
+    - Use a lock.amount smaller then its current capacity. If the amount is
+      higher then the recipient will be reject it, as it means spending money
```

Comment:
May you rephrase this line, please?

Reply:
I think it's understandable... what exactly do you find that needs rephrasing?

Reply:
`will be` can't be followed by verb in the present tense. Maybe what he meant was `will reject it`? Also, if it's like that, then must be preceeded by comma: `If the amount is higher, then...`

Reply:
I think it's just a typo. He wanted to write: `will be rejected`